packages/webcomponents/src/components/payments-list/fetch-payments-response.type.ts

```diff
@@ -0,0 +1,96 @@
+interface PageInfo {
+  has_previous: boolean;
+  has_next: boolean;
+  start_cursor: string;
+  end_cursor: string;
+}
+
+interface Card {
+  id: string;
+  acct_last_four: string;
+  brand: string;
+  name: string;
+  token: string;
+  metadata: null;
+  created_at: string;
+  updated_at: string;
+}
+
+interface BankAccount {
```

Comment:
I believe we have a lot of these types already and they will be needed for other components, does it make sense to put the ones that are missing in the files in the `api` directory?